hrm-domain/hrm-core/profile/sql/profile-get-sql.ts

```diff
@@ -76,10 +80,14 @@ export const getProfilesSqlBase = (selectTargetProfilesQuery: string) => `
   LEFT JOIN LATERAL (SELECT COUNT(*) > 0 as "hasContacts", "profileId", "accountSid" FROM "Contacts" c WHERE profiles.id = c."profileId" AND profiles."accountSid" = c."accountSid" GROUP BY "profileId", "accountSid") hrc ON true
 `;
 
-export const getProfileByIdSql = getProfilesSqlBase(`
+export const getProfileByIdSql = (includeSectionContent: boolean) =>
```

Comment:
I would have thought the relevant info for profiles flags and identifiers should be included. Wondering if they is a reason this is not required in scope of profile info

Reply:
I think there are some fields of the profile flags that are missing. I don't remember from the top but you'll probably figure it out when you have to define the schema and tables for the Data Warehouse.